DataStructureClasses/addContact.ts

```diff
@@ -0,0 +1,50 @@
+import ContactNotFoundError from "./error";
+
+
+export default class PhoneBook {
+    contacts: any[];
```

Comment:
We should not use **any** as per the best practices, give appropriate types. Implement it everywhere.

Reply:
1) Typescript type checker doesn't perform type checking of variables
2) it is difficult to understand which type of data it hold 
3) when we got the error it difficult to debug if project is complex